README.md

```diff
@@ -15,3 +15,9 @@ The keyed text format (`ktx`) is a minimal human readable key-values to store te
 
 This work is licensed under the MIT license.  
 [![DOI](https://zenodo.org/badge/10173/rougier/numpy-100.svg)](https://zenodo.org/badge/latestdoi/10173/rougier/numpy-100)
+
+
+### Variants in Other Languages
```

Comment:
Maybe a single "Julian variant" would be better (until we get some other variants)

Reply:
Yep, that will work.

Sorry for not stating the author.
Obvious on my side but I should have.